tests/integration/targets/copr/tasks/main.yml

```
@@ -0,0 +1,65 @@
+---
+- block:
```

Assuming running these tests only makes sense with Fedora:
```suggestion
- when: ansible_distribution == 'Fedora'
  block:
```